content/docs/1.6.0/volumes-and-nodes/storage-class-parms.md

```diff
@@ -0,0 +1,95 @@
+Type:  Boolean  
+Notes:  If enabled, then use the following additional parameters to specify image details.  Backing images are a whole topic to themselves - see [Backing Images](../../advanced-resources/backing-images)  
+- **Backing Image Name**  
+  Specify the name of a backing image resource that will be used by Longhorn volumes made with this storage class.  
+  Type:  String  
+  Notes:  If the image does not exist, the backing image data source type and backing image data source parameters should be specified so that Longhorn will create the backing image before using it.  
+- **Backing Image Checksum**  
+  Specify the expected SHA512 checksum of the selected existing backing image.  
+  Type: String  
+  Notes:
+  - If the backing image name is not specified, setting this field is meaningless.
+  - It is not recommended to set this field if the data source type is \"export-from-volume\".'
+- **Backing Image DataSource Type**  
+  Specify the data source type for the backing image used in Longhorn StorageClass.  
+  Type:  Enumerated string, one of "" (default), "download", "upload", or "export-from-volume".  
+  Notes:  
+  - If the backing image name is not specified, setting this field is meaningless.
+  - If the backing image does not exist, Longhorn will use this field to create one.  Otherwise, Longhorn will use it to verify the selected backing image.
+  - For backing image creation with data source type \"upload\", it is recommended to do it via UI rather than StorageClass here. Uploading requires file data sending to the Longhorn backend after the object creation, which is complicated if you want to handle it manually.  
+- **Backing Image DataSource Parameters**  
+  Specify the data source parameters for the backing image to be used.  
+  Type:  String with JSON map. e.g., '{\"url\":\"https://backing-image-example.s3-region.amazonaws.com/test-backing-image\"}'.  
+  Notes:
+  - If the backing image name is not specified, setting this field is meaningless.
+  - If the backing image does not exist, Longhorn will use this field to create one.  Otherwise, Longhorn will use it to verify the selected backing image.
+  - Be careful of the quotes here.
+
+### Remove Snapshots During Filesystem Trim
+Allow automatically removing snapshots during filesystem trim for this Longhorn storage class.  
+Type: Enumerated string, one of "ignored" (default), "enabled", or "disabled".  
+Notes:  This seems like it overlaps the global setting `Remove Snapshots During Filesystem Trim`, and it does.  The idea is the same, but one key difference is that the global is a Boolean, and this one an enumeration.  This one sets a policy for all volumes created with this storage class.  The setting "enabled" means always allow it, "disabled" means never allow it, and "ignored" (the default) means the global setting should be used.   See more in [Trim Filesystem](../trim-filesystem).
```

Comment:
You can add a refer link to the setting. Then there is no need to explain how this field works.